packages/affiliates/gas-rebate/VoterGasRebate.js

```diff
@@ -558,6 +566,9 @@ async function getHistoricalGasPrice(startBlock, endBlock) {
 
     let data = (await response.json()).result;
 
+    if (!Array.isArray(data))
+      throw new Error("Unexpected ETH gas price historical data, did you set ETHERSCAN_API_KEY environment variable?");
```

Comment:
do you just want to assert for `ETHERSCAN_API_KEY` before running script, or is it optional?

Reply:
oh good call! its definitely required

Reply:
I realize that I actually do this but its possible to set the incorrect api key, so i'll edit this error to just explain the data is ill formatted